livekit_egress.proto

```diff
@@ -32,31 +32,56 @@ message RoomCompositeEgressRequest {
   bool audio_only = 3;          // (default false)
   bool video_only = 4;          // (default false)
   string custom_base_url = 5;   // template base url (default https://recorder.livekit.io)
-  oneof output {                // required
-    EncodedFileOutput file = 6;
-    StreamOutput stream = 7;
-    SegmentedFileOutput segments = 10;
+  oneof output {                // deprecated
+    EncodedFileOutput file = 6 [deprecated = true];
+    StreamOutput stream = 7 [deprecated = true];
+    SegmentedFileOutput segments = 10 [deprecated = true];
   }
   oneof options {
     EncodingOptionsPreset preset = 8; // (default H264_720P_30)
     EncodingOptions advanced = 9;     // (optional)
   }
+  EncodedFileOutput file_output = 11;
```

Comment:
do we want to allow multiple of these too? i.e. write to two different file outputs? might be overkill?

Reply:
probably overkill, since the encoding parameters and output codecs need to be the same. Would just be making 2 copies of the same file

Reply:
How about the case where we'd want the file uploaded to more than 1 place?